src/platforms/common/enriching-events/breadcrumbs.mdx

```diff
@@ -25,7 +25,13 @@ You can manually add breadcrumbs whenever something interesting happens. For exa
 
 <PlatformContent includePath="breadcrumbs-example" />
 
-The available breadcrumb keys are `type`, `category`, `message`, `level`, `timestamp` (which many SDKs will set automatically for you), and `data`, which is the place to put any information you'd like the breadcrumb to include. Using keys other than these six won't cause an error, but will result in the data being dropped when the event is processed by Sentry.
+<!-- TODO: Native shouldn't really be on this list, since c++ is statically typed. Remove it once types have been added to the native SDK. -->
```

Comment:
The Native SDK will never receive types, so it needs to stay here.

Reply:
> For native, we can show it since we haven't typed out the protocol.

Ah - for some reason I read an implied "yet" in there. Will take out the todo.